lib/capybara/spec/session/attach_file_spec.rb

```diff
@@ -11,33 +11,33 @@
 
   context "with normal form" do
     it "should set a file path by id" do
-      @session.attach_file "form_image", __FILE__
+      @session.attach_file "form_image", attachment_path(__FILE__)
```

Comment:
please change the name of `attachment_path` to mention `os`so it's clear it's modifying the file names in an OS specific manner